bin/node/cli/src/command.rs

```diff
@@ -70,11 +71,12 @@ pub fn run() -> Result<()> {
 	match &cli.subcommand {
 		None => {
 			let runner = cli.create_runner(&cli.run)?;
-			runner.run_node(
-				service::new_light,
-				service::new_full,
-				node_runtime::VERSION
-			)
+			runner.print_node_infos(node_runtime::VERSION);
```

Comment:
Should it be there or already included in `run_node_until_exit`? :thinking: 

...since sc-cli initialize the logger itself when `create_configuration` is ran, maybe it would make sense to let sc-cli print those infos (in `run_node_until_exit` for instance).

I guess I have a slight preference for that but both are good and I'm not sure at all tbh. It's already so perfect!

Reply:
Yeah that might be the better option here. I'm still trying to work out where the boundaries between crates should be!

Reply:
I think in that case `print_node_infos` should be internal and not public.